tests/network/localnet/liblocalnet.py

```diff
@@ -1,12 +1,57 @@
+from collections.abc import Generator
+from contextlib import contextmanager
+
+import tests.network.libs.nodenetworkconfigurationpolicy as libnncp
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
+from tests.network.libs.nodenetworkconfigurationpolicy import Resource
+from utilities.constants import OVS_BRIDGE
 
 NETWORK_NAME = "localnet-network"
+_IPERF_SERVER_PORT = 5201
+
+
+@contextmanager
+def start_server_vm(vm: BaseVirtualMachine) -> Generator[Server]:
+    with Server(vm=vm, port=_IPERF_SERVER_PORT) as server:
+        assert server.is_running()
+        yield server
+
+
+@contextmanager
+def start_client_vm(vms: tuple[BaseVirtualMachine, BaseVirtualMachine]) -> Generator[Client]:
+    vm_server, vm_client = vms
+    with Client(
+        vm=vm_client,
+        server_ip=lookup_iface_status(vm=vm_server, iface_name=NETWORK_NAME)[IP_ADDRESS],
+        server_port=_IPERF_SERVER_PORT,
+    ) as client:
+        assert client.is_running()
+        yield client
+
+
+def additional_ovs_bridge_interface(bridge_name: str, worker_port_name: str) -> libnncp.Interface:
```

Comment:
Why this needs to appear in the libs? I mean, why it is not inline in the configuration? To me, seeing the whole configuration/spec in one place is more clear on what is going on.

The name also is odd... this returns an ovs-bridge interface. If it is in addition to some other interface or not, is not known to this thing. I mean, you do not know how it will be used.

Reply:
I agree that this does not need to be in a separate function; it just wraps `libnncp.Interface`